src/app/pages/permissions-page/permissions-page.page.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Router } from '@angular/router';
+
+import { ProfileModelService } from '../profile-page/_services/profile.model.service';
+
+import {AuthService} from "@savvato-software/savvato-javascript-services";
+
+import * as moment from 'moment'
+import {LoadingService} from "../../_services/loading-spinner/loading.service";
+import {MenuController} from "@ionic/angular";
+
+@Component({
+  selector: 'app-permissions-page',
+  templateUrl: './permissions-page.page.html',
+  styleUrls: ['./permissions-page.page.scss'],
+})
+
+
+export class PermissionsPagePage implements OnInit {
+
+  constructor(private router: Router,
+    private _profileModelService: ProfileModelService,
```

Comment:
Anywhere you see Profile, you need to complete your copy and paste, and change Profile to Permissions, as appropriate.